python/tvm/ir/module.py

```diff
@@ -255,3 +255,18 @@ def __str__(self):
 
     def __repr__(self):
         return self.astext()
+
+    def script(self, show_meta=False) -> str:
+        """Print IRModule into TVMScript
+
+        Parameters
+        ----------
+        show_meta : bool
+            Whether show meta
```

Comment:
I've always been curious that what "meta information" means for TVM script.

Besides, would be great to say it more completely.
```suggestion
            Whether showing the script meta information
```